python/tank/constants.py

```diff
@@ -83,7 +83,7 @@
 TOOLKIT_INIT_CACHE_FILE = "toolkit_init.cache"
 
 # URL for contacting support
-SUPPORT_URL = "https://support.shotgunsoftware.com"
+SUPPORT_URL = "https://developer.shotgridsoftware.com"
```

Comment:
This one is tricky. That variable is used in different places and it is mostly mean to contact support. I think we might need to use this URL: https://help.autodesk.com/view/SGSUB/ENU/

What do you think?

Reply:
https://knowledge.autodesk.com/contact-support